cli/download.go

```diff
@@ -10,51 +10,61 @@ func download(c *cli.Context) error {
 	// Input validation
 	collectionURI := c.String(CollectionKey)
 	if collectionURI == "" {
-		log.Fatalln("[STEPMAN] - No step collection specified")
+		log.Fatalf("No step collection specified")
 	}
 	route, found := stepman.ReadRoute(collectionURI)
 	if !found {
-		log.Fatal("No route found for lib: " + collectionURI)
+		log.Fatalf("No route found for lib: %s" + collectionURI)
```

Comment:
`"No route found for lib: %s" + collectionURI` -> `"No route found for lib: %s", collectionURI`